packages/jaeger-ui/src/utils/version/get-version.test.js

```diff
@@ -34,9 +34,9 @@ describe('getVersion()', () => {
     console.warn = oldWarn;
   });
 
-  describe('`window.getVersion` is not a function', () => {
+  describe('`window.getJaegerVersion` is not a function', () => {
```

Comment:
what is the origin of this change? I only see the test change, not the main code.

Reply:
It looks like this test was testing for the wrong function name - it should be `window.getJaegerVersion` as per https://github.com/jaegertracing/jaeger-ui/blob/dda3b97edccac392324ddfc4d90487377c1ca47e/packages/jaeger-ui/src/utils/version/get-version.tsx.